scripts/pre-push

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# INSTRUCTIONS:
```

Comment:
These don't seem like instructions? Suggest removing this.

Reply:
done